roles/spads/tasks/initialSetup.yaml

```diff
@@ -12,6 +12,19 @@
     name: "*"
     state: latest
 
+- name: Copy using inline content
+  ansible.builtin.copy:
+    content: 'Etc/UTC\n'
+    dest: /etc/timezone
+    owner: root
+    group: root
+    mode: '0644'
```

Comment:
I don't believe we need to set the owner, group and mode: this is already executed as root, so no need to set explicitly.